erdkotlin/src/main/java/com/elrond/erdkotlin/domain/transaction/models/Transaction.kt

```diff
@@ -69,4 +73,19 @@ data class Transaction(
         private val gson = GsonBuilder().disableHtmlEscaping().create()
     }
 
+    enum class Version {
+        DEFAULT,
```

Comment:
this isn't right. The `version` and `options` pairs are as follows:
1. normal transaction: version=1; options=0
2. transaction signed with hash: version=2; options=1
please refer to this constants:
https://github.com/ElrondNetwork/elrond-sdk/blob/576fdc4bc0fa713738d8556600f04e6377c7623f/erdjs/src/constants.ts

Reply:
Since we use `version.serializedValue` (line 56) and `option.serializedValue` (line 58) the code is doing exactly what you are describing.
Version.DEFAULT is translated to 1
Version.TX_HASH_SIGN is translated to 2
Option.NONE is translated to 0 (and isn't serialized in that case)
Option.TX_HASH_SIGN is translated to 1

However, your comment is showing that the code is not clear enough.
I will replace both enum with int constants.

Reply:
Done